### DISCUSSION TITLE
Exhaust report request submit API logs.

### DISCUSSION THREAD

**Discussion by vatsaraj** [Sunbird-Lern/Community#79] [Query] [answered]
Are the exhaust report request API - /api/dataset/v1/request/submit - payload and response stored anywhere?
The use-case is where an actor wants to ascertain how many data request submit API calls were made, in, say, the last 3 weeks, how many such submit requests succeeded, how many failed and what were the payload and responses.

**Comment by reshmi-nair** [comment, marked as answer]
@vatsaraj Exhaust API is part of Observ. Pls find the documentation for the same. [Exhaust APIs](https://obsrv.sunbird.org/learn/product-and-developer-guide/data-service)

**Reply by vatsaraj** [reply]
@reshmi-nair sure, I'll check that one. Thanks.
